### PR TITLE
Harshpavuluri/vickrey auction addition

### PR DESCRIPTION
**Pull request by harshpavuluri**
Included new sidebar menu for Vickrey auctions. Added a new MD file for vickrey info. Mostly copy and paste from the Keycult website with some additions to make it more general. Added FAQ section to add to condense section. Listed previous pricing from Obby's post and repercussions for not paying

### CODE REVIEW

**Comment by harshpavuluri** [comment]
closing because main site is being updated

**Comment by harshpavuluri** [comment]
Reopen for Obby changes

File: pages/mydoc/vickrey_info_rules.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Vickrey Auction Information
+You can find a detailed explaination about the Vickrey format and FAQs on the main website here: <a href="https://keycult.com/blogs/typings/the-vickrey-auction-and-why-were-using-it">Vickrey Auction Information</a>.
+
+
+## General Vickery Rules
+
+These rules apply to all vickrey auctions conducted by Keycult. Failure to abide by them may get you disqualified, banned, or blacklisted from future vickreys, raffles and/or other releases.
+
+ - You have 10 hours to enter on auction day (10am to 8pm EDT) and cannot change your bid after you submit! Please be sure of your submission and take your time.
+ - You are only allowed to bid once. Multiple bids are not allowed nor will help you.
```
Comment (ObsidianRaid) [inline pages/mydoc/vickrey_info_rules.md:19]: We actually disqualify anyone who bids more than once, so we might want to spell that out here

File: pages/mydoc/vickrey_info_rules.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ - You may not bid on behalf of someone else or agree ahead of time to sell your slot to someone else if you win.
+ - If you win, you will receive an invoice through our Shopify storefront and will have 24 hours to complete payment. **We do not accept PayPal.**
+
```
Comment (ObsidianRaid) [inline pages/mydoc/vickrey_info_rules.md:22]: Should we also include something about failure to pay will result in a ban from all future keycult sales (raffles, auctions, etc.)

**Comment by harshpavuluri** [comment]
Rules included. I also a bit of flexible language to give the moderators a range of options depending on infraction severity

File: pages/mydoc/vickrey_info_rules.md [hunk 3]
```diff
@@ -16,9 +16,10 @@ You can find a detailed explaination about the Vickrey format and FAQs on the ma
 These rules apply to all vickrey auctions conducted by Keycult. Failure to abide by them may get you disqualified, banned, or blacklisted from future vickreys, raffles and/or other releases.
 
  - You have 10 hours to enter on auction day (10am to 8pm EDT) and cannot change your bid after you submit! Please be sure of your submission and take your time.
- - You are only allowed to bid once. Multiple bids are not allowed nor will help you.
+ - You are only allowed to bid once. Multiple bids in any form will result in a immediate **disqualification** from the current and/or future sales.
  - You may not bid on behalf of someone else or agree ahead of time to sell your slot to someone else if you win.
  - If you win, you will receive an invoice through our Shopify storefront and will have 24 hours to complete payment. **We do not accept PayPal.**
+    - Failure to pay will result in a **ban** from all future Keycult sales, including but not limited to the following: raffles, auctions, etc.
```
[the remark this answers is not in the document]
Comment (harshpavuluri) [reply pages/mydoc/vickrey_info_rules.md:22]: Nope. Just a sub-bullet for the point on winning. Looked a bit better that way

[the remark this answers is not in the document]
Comment (ObsidianRaid) [reply pages/mydoc/vickrey_info_rules.md:22]: Ahh okay sounds good!

**Pull request merged**